LibVLCSharp/Shared/LibVLC.cs

```diff
@@ -531,6 +531,14 @@ void UnsetLog()
 
         #region DialogManagement
 
+        private static readonly DisplayErrorCallback InternalErrorCallback = Error;
+        private static readonly DisplayLoginCallback InternalLoginCallback = Login;
+        private static readonly DisplayQuestionCallback InternalQuestionCallback = Question;
+        private static readonly DisplayProgressCallback InternalDisplayProgressCallback = DisplayProgress;
+        private static readonly CancelCallback InternalCancelCallback = Cancel;
+        private static readonly UpdateProgressCallback InternalUpdateProgressCallback = UpdateProgress;
+        private static readonly DialogCallbacks DialogCbs = new DialogCallbacks(InternalErrorCallback, InternalLoginCallback, InternalQuestionCallback, InternalDisplayProgressCallback, InternalCancelCallback, InternalUpdateProgressCallback);
```

Comment:
Why not new the struct `SetDialogHandlers`?

Reply:
The struct will never change since it uses static callbacks. That's why.
Maybe we don't need to keep the struct once it's passed to the method? But I didn't want to change the behavior too much.